django_prices/models.py

```diff
@@ -114,6 +118,16 @@ def __set__(self, instance, value):
         setattr(instance, self.net_field, net)
         setattr(instance, self.gross_field, gross)
 
+    def __eq__(self, other):
+        if isinstance(other, Field):
```

Comment:
I think we should also handle the case where `other` is another instance of `TaxedMoneyField`. Same applies to `__lt__`.